src/resources/js/tickets.js

```diff
@@ -31,8 +37,24 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 	// misc ticket elements
 	var $ticket_image_preview            = $( document.getElementById( 'tribe_ticket_header_preview' ) );
 	var $ticket_show_description         = $( document.getElementById( 'tribe_tickets_show_description' ) );
-	var date_format                      = 'YYYY-MM-DD';
-	var time_format                      = 'HH:mmA';
+
+	// Datepicker and Timepicker variables
+	var datepicker_formats = [
+		'yy-mm-dd',
+		'm/d/yy',
+		'mm/dd/yy',
+		'd/m/yy',
+		'dd/mm/yy',
+		'm-d-yy',
+		'mm-dd-yy',
+		'd-m-yy',
+		'dd-mm-yy',
+		'yy.mm.dd',
+		'mm.dd.yy',
+		'dd.mm.yy'
+	];
+	var dateFormat = datepicker_formats[ 0 ];
```

Comment:
Don't need to change it necessarily, but it's weird that we're using camelCase and snake_case for different vars.